sample/allofhands_hawaii2016/sessions

```diff
@@ -0,0 +1,3723 @@
+[
+
+	{
+		"short_abstract": "",
+		"speakers": [],
+		"subtitle": null,
+		"language": "english",
+		"title": "Hawaii All Hands - Plenary Session",
+		"track": {
+			"id": 1,
+			"title": "All Hands"
+		},
+
+		"start_time": "2016-12-06T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-06T21:00:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 19,
+			"name": "Monarchy Ballroom"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 7
+	},
+
+
+	{
+		"short_abstract": "",
+		"speakers": [{
+			"id": 105,
+			"name": "Sean White",
+			"organisation": null
+		}],
+		"subtitle": null,
+		"language": "english",
+		"title": "IT: Application Services - Service Strategy Pivot",
+		"track": {
+			"id": 3,
+			"title": "IT"
+		},
```

Comment:
@mahikaw  There are many sessions which have the track name within the title itself. I feel that it is redundant information as the track tags are going to be displayed in the schedule page anyway. And If we remove the track name from the session title we could display more text from the actual topic in the `TextView`.

Above we see that 'IT: Application Services - Service Strategy Pivot'
I guess 'Application Services - Service Strategy Pivot' this would suffice. There are many instances of this in the file.
@arp95 @heysadboy  Opinions?